pom.xml

```diff
@@ -4,40 +4,23 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>org.octopusden.octopus</groupId>
+        <artifactId>octopus-parent</artifactId>
+        <version>2.0.6</version>
+    </parent>
+
     <groupId>org.octopusden.octopus.cdt</groupId>
     <artifactId>MCSDistributionBuilder</artifactId>
-    <version>02.00</version>
-
-    <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-    </properties>
```

Comment:
@CamaradeRoman , I am not sure this change did not break the build outside *github* (manual from source on my workstation for example).
Please disprove if I'm wrong.